src/components/Meme.js

```diff
@@ -81,6 +67,77 @@ export default function Meme() {
 		}
 	}
 
+	//data for all the input texts that are supposed to be draggable
+	const [memeTextData, setMemeTextData] = useState(
+		[
+			{
+				id: 1,
+				draggable: true,
+				style: {
+					position: "absolute",
+					left: "50%",
+					transform: "translateX(-50%)",
+					top: "0px",
+				}
+			},
+			{
+				id: 2,
+				draggable: true,
+				style: {
+					position: "absolute",
+					left: "50%",
+					transform: "translateX(-50%)",
+					top: "",
+					bottom: "0px"	
+				}
+			}
+		]
+	)
+	
+	//redering the text data in h2 tag 
+	const memeTexts = memeTextData.map(text => {
+		return <h2
+				key={text.id}
+				draggable={text.draggable}
```

Comment:
draggable will always be true for all text, then why are you taking it from object